.github/workflows/e2e-k3d.yml

```
@@ -47,6 +47,13 @@ jobs:
           curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | bash
           k3d version
       - uses: azure/setup-helm@v4
+      - name: Wait for submitToEnaProduction check
+        uses: lewagon/wait-on-check-action@v1.3.4
+        with:
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+          check-name: Validate submitToEnaProduction is not true in values.yaml
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          wait-interval: 10
```

```suggestion
          wait-interval: 1
```

I don't like this wait-on-check-action, it's stupidly slow to setup, large overhead, but I guess we can use for now

---

Why do we need to fail in e2e-k3d? Ingest is not used in e2e so we don't need to wait for it?

---

we can remove it in e2e - its most important in the ena-submission-image